### PR TITLE
Add ChainerCI config

### PR DESCRIPTION
**Pull request by disktnk**
~(draft, don't merge this)~

- ~uploading docker image, base for CI~ -> done, `disktnk/chainer-compiler:ci-cuda100-ngraph0.19.0-tvm0.5`
- ~"runtests" fails~ -> set `-g` and `--ngraph`
  - need to amend #280 error to succeed

### CODE REVIEW

File: .chainerci/run-chainerci-tests.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    -DCHAINERX_BUILD_PYTHON=ON \
+    && \
+    ninja
+
+make large_tests
+
+make test
+cd ..
+
+python3 scripts/runtests.py -G --ngraph
```
Comment (shinh) [inline .chainerci/run-chainerci-tests.sh]: FYI, setting `-G` and `--ngraph` at once does not make much sense (I don't think ngraph op can handle cuda ndarray). I think CI should run runtests.py several times for each backend.

Comment (disktnk) [reply .chainerci/run-chainerci-tests.sh]: I split those conditions, thx

```
python3 scripts/runtests.py -g
python3 scripts/runtests.py --ngraph
```

**Comment by disktnk** [comment]
ChainerCI, test this please

**Comment by disktnk** [comment]
ChainerCI, test this please

**Pull request merged**